inc/namespace.php

```diff
@@ -922,8 +922,9 @@ function get_object_cache_stats() : array {
  * @return array
  */
 function redact_metadata( $metadata ) {
+	global $xray_redact_keys;
 
-	$redact_keys_default = [];
+	$redact_keys_default = (array) ( $xray_redact_keys ?? [] );
```

Comment:
@roborourke Though this works, but I feel there is potential downside with the use of global variable here.
For example, it could be overridden accidentally by other code.
I'm thinking to use constant here or composer env instead? So other engineers forced to add their key from one place.

Reply:
A constant would have the issue of preventing other code adding to it. It would be up to application developers to check for existing keys etc... This is a pattern pretty common in WP like `$batcache` for instance. It isn't ideal but I think it's the best balance between flexibility, extensibility and ease of use.

If you wanted to add to this from a plugin for instance you'd use:

```php
global $xray_redact_keys;

$xray_redact_keys = array_merge_recursive( $xray_redact_keys ?? [], [
  '$_POST' => [ '... my custom keys ...' ],
] );
```

This can be documented in the Altis docs.